### PR TITLE
Update clause_8_collections.adoc

### PR DESCRIPTION
**Pull request by chris-little**
mainly spelling typos, some grammar.

### CODE REVIEW

File: standard/clause_8_collections.adoc [hunk 1]
```diff
@@ -230,11 +230,11 @@ A template for the definition of the parameter in YAML according to OpenAPI 3.0
 [[collection-general-requirements]]
 === General Requirements
 
-The following general requirements and recomendations apply to all OGC APIs which host Spatial Resources.
+The following general requirements and recommendations apply to all OGC APIs which host Spatial Resources.
 
 ==== Coordinate Reference Systems
 
-As discussed in Chapter 9 of the W3C/OGC Spatial Data on the Web <<SDWBP,Best Practices document>>, how to express and share the location of resources in a consistent way is one of the most fundamental aspects of publishing geographic data and it is important to be clear about the coordinate reference system that coordinates are in.
+As discussed in Chapter 9 of the W3C/OGC Spatial Data on the Web <<SDWBP,Best Practices document>>, how to express and share the location of resources in a consistent way is one of the most fundamental aspects of publishing geographic data and it is important to be clear about the coordinate reference system that define the coordinates.
```
Comment (bradh) [inline standard/clause_8_collections.adoc]: The CRS doesn't really define the coordinates - it just helps the interpretation. I agree the original words weren't ideal though. A friend also has a pet peeve about "it is important to" language that doesn't say *why* it is important, which might be valid here too.

How about something like:

> As discussed in Chapter 9 of the W3C/OGC Spatial Data on the Web <<SDWBP,Best Practices document>>, a fundamental aspect of publishing geographic data is how to express and share the location of resources in a consistent way, including the coordinate system(s) that are used.

Comment (cmheazel) [reply standard/clause_8_collections.adoc]: @chris-little I have to agree with @bradh. The original text is poorly written, but I don't think the change captures what we want to say.  Lets leave this PR open while we try to come up with better wording.

Comment (chris-little) [reply standard/clause_8_collections.adoc]: @cmheazel I like @bradh 's words as they reflect the W3C SDW BP document: "One of the most fundamental aspects of publishing spatial data, data about location, is how to express and share the location in a consistent way". I also think we are trying to condense three W3C SDW BPs:
 Best Practice 6: Provide geometries at the right level of accuracy, precision, and size
 Best Practice 7: Choose coordinate reference systems to suit your user's applications
 Best Practice 8: State how coordinate values are encoded. 

How about:
"As discussed in Chapter 9 of the W3C/OGC Spatial Data on the Web <<SDWBP,Best Practices document>>, a fundamental aspect of publishing geographic information is how to express and share the location of resources in an appropriate and consistent way, including the coordinate system(s) that are used."

Comment (cmheazel) [reply standard/clause_8_collections.adoc:237]: @chris-little API-Common has undergone a serious re-write.  I believe the new version captures these changes.  Let me know if you agree.  If so, then we can close this PR.
The new version is in the April-202-Updates branch.

Comment (chris-little) [reply standard/clause_8_collections.adoc:237]: Agreed

**Comment by chris-little** [comment]
Looks good to me too.

**Comment by dblodgett-usgs** [comment]
The typos no longer appear in the repository.

**Pull request closed without merging**